tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Log4Net/LogsInjection/AppenderAttachedImplIntegration.cs

```diff
@@ -19,7 +19,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Log4Net
         ReturnTypeName = ClrNames.Int32,
         ParameterTypeNames = new[] { "log4net.Core.LoggingEvent" },
         MinimumVersion = "1.0.0",
-        MaximumVersion = "2.*.*",
+        MaximumVersion = "3.*.*",
```

Comment:
I am planning on doing the due diligence to verify that this is correct still.

Reply:
Appears to be good still, there are some new nullable checks in here added, but should be fine.

https://github.com/apache/logging-log4net/blob/8bf4313cdc7dc2bcc14583c8b206c5a25b7edf4a/src/log4net/Util/AppenderAttachedImpl.cs#L64-L91